includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -335,6 +336,7 @@ public function pull( $items ) {
 			update_post_meta( $new_post, 'dt_original_post_url', esc_url_raw( $post_array['link'] ) );
 			update_post_meta( $new_post, 'dt_original_site_name', sanitize_text_field( $post_array['original_site_name'] ) );
 			update_post_meta( $new_post, 'dt_original_site_url', sanitize_text_field( $post_array['original_site_url'] ) );
+			update_post_meta( $new_post, 'dt_original_post_parent', (int) $original_post_parent );
```

Comment:
A check for `$original_post_parent !== 0` before adding the custom field would be good, so as not to store unnecessary data.  The field not being present would imply that the origin post did not have a parent.